docs-conceptual/azps-11.6.0/authenticate-interactive.md

```diff
@@ -22,55 +22,127 @@ principals or other noninteractive authentication methods.
+[1]   Facility Services Subscription        00000000-0000-0000-0000-000000000000      Contoso
+[2] * Finance Department Subscription       00000000-0000-0000-0000-000000000000      Contoso
+[3]   Human Resources Subscription          00000000-0000-0000-0000-000000000000      Contoso
+[4]   Information Technology Subscription   00000000-0000-0000-0000-000000000000      Contoso
+
+The default is marked with an *; the default tenant is 'Contoso' and subscription is 'Finance Department Subscription (00000000-0000-0000-0000-000000000000)'.
+
+Select a tenant and subscription (type a number or Enter to accept default): 4
+
+Subscription name                       Tenant name
+------------------------------------    --------------------------
+Information Technology Subscription     Contoso
+```
+
+Commands run against this subscription by default. To change your active subscription, use the
+`Set-AzContext` cmdlet. For more information, see
+[Azure PowerShell context objects][context-persistence].
+
+Use `Update-AzConfig` to set your default subscription and prevent being prompted to select a
+subscription each time you log in interactively.
+
+```powershell
+Update-AzConfig -DefaultSubscriptionForLogin '<subscription name or id>'
+```
+
+## Web Account Manager (WAM)
+
+Beginning with Az PowerShell module version 12.0.0, Azure PowerShell's default login authentication
+method for Windows-based systems is Web Account Manager (WAM).
+
+WAM is a Windows 10+ component that acts as an authentication broker. An authentication broker is an
+application that runs on a user's machine that manages the authentication handshakes and token
+maintenance for connected accounts.
+
+### Benefits of WAM
+
+Using WAM offers several benefits:
+
+- Enhanced security. See
+  [Conditional Access: Token protection (preview)][token-protection].
+- Support for Windows Hello, conditional access policies, and FIDO keys.
+- Streamlined single sign-on.
+- Bug fixes and enhancements shipped with Windows.
 
-   ```azurepowershell
-   Connect-AzAccount
-   ```
+### Limitations of WAM
 
-   If Azure PowerShell can open your default browser, it initiates
-   [authorization code flow][authorization-code-flow] and opens the default browser to load an Azure
-   sign-in page. Otherwise, it initiates [device code flow][device-code-flow] which instructs you to
-   open a browser page at [microsoft.com/devicelogin][ms-devicelogin] and enter the code displayed
-   in your PowerShell session.
+At the current stage of development, there are a few known limitations to WAM:
+
+- WAM is available on Windows 10 and later and on Windows Server 2019 and later. On earlier versions
+  of Windows, Linux, and macOS, Azure PowerShell automatically defaults to a browser.
+- Microsoft Accounts (for example, @outlook.com or @live.com) aren't currently supported. We're
+  working with the Microsoft Identity team to bring this support later.
```

Comment:
Currently, to login Microsoft Accounts with WAM, please specify tenant Id as the following script does so that you won't be blocked by MFA
```
Connect-AzAccount -Tenant $tenantId
```